<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fontify</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/icons/type.png</normaloff>:/icons/type.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 41, 41);
QLabel{
	color:white;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color:white;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>80</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>80</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	box-shadow: 2px 2px 1px 1px #E9E9E9;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../icons/icons.qrc">:/icons/type.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>Cantarell</family>
                 <pointsize>34</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Fontify</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight|Qt::AlignTop">
              <widget class="QFrame" name="theme_togle_frame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="../icons/icons.qrc">:/icons/themes-white.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                  <property name="margin">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="switch_theme">
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QCheckBox::indicator{
	width:30px;
	height:30px;
}

QCheckBox::indicator:uncheked{
	image:url(&quot;:/icons/switch-on-white.png&quot;)
}

QCheckBox::indicator:checked{
	image:url(&quot;:/icons/switch-off-white.png&quot;)
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="../icons/icons.qrc">:/icons/theme-white.png</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="fontInput">
               <property name="minimumSize">
                <size>
                 <width>450</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>450</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
	color: rgb(154, 153, 150);
}</string>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Font to install</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Ignored</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>22</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="openDialog">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color: rgb(36, 31, 49);
	color:white;
	border-style:solid;
	border-width:1px;
	border-color:black;
	text-align:center;
	padding:4px;
	border-radius:4px;
}

QPushButton:hover{
	background-color: rgb(61, 56, 70);
}</string>
               </property>
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="messageLabel">
            <property name="font">
             <font>
              <family>Cantarell</family>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="margin">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>185</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="installButton">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(38, 162, 105);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Install</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>184</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="previewFrame">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color:white;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="display1">
            <property name="font">
             <font>
              <pointsize>25</pointsize>
             </font>
            </property>
            <property name="text">
             <string>ABCDE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="display2">
            <property name="font">
             <font>
              <pointsize>34</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>ABCDE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="display3">
            <property name="font">
             <font>
              <pointsize>48</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>ABCDE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="versionlabel">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color:white;
}</string>
               </property>
               <property name="text">
                <string>Version 1.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMore">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>More</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>true</bool>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionBackup"/>
    <addaction name="actionReport_Bug"/>
   </widget>
   <addaction name="menuMore"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionBackup">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/backup.png</normaloff>:/icons/backup.png</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/icons/beetle.png</normaloff>:/icons/beetle.png</iconset>
   </property>
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
  <include location="../../../../../../.designer/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
